src/clj/game/core/actions.clj

```diff
@@ -35,6 +35,7 @@
   (when (and (not (get-in @state [side :register :cannot-draw]))
              (pay state side nil :click 1 {:action :corp-click-draw}))
     (system-msg state side "spends [Click] to draw a card")
+    (trigger-event state side (if (= side :corp) :corp-pre-click-draw :runner-pre-click-draw) nil)
```

Comment:
lol If you're adding a new one, might as well make it `trigger-event-simult`? Future proof ourselves!